### PR TITLE
Listen to ipv6 on default

### PR DESCRIPTION
**Pull request by t2d**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ Defaults:
 
     sshd_port: 22
     sshd_listen_address: 0.0.0.0
+    sshd_listen_address_ipv6: ::
```
Comment (doobry-systemli) [inline README.md]: Another approach:

```
# Set `sshd_listen_addresses` to a list of addresses. Defaults to all IPv4+IPv6 addresses if unset.
#sshd_listen_addresses: []
```

File: templates/sshd_config.j2 [hunk 2]
```diff
@@ -4,8 +4,8 @@
 # What ports, IPs and protocols we listen for
 Port {{ sshd_port }}
 # Use these options to restrict which interfaces/protocols sshd will bind to
-#ListenAddress ::
 ListenAddress {{ sshd_listen_address }}
+ListenAddress {{ sshd_listen_address_ipv6 }}
```
Comment (doobry-systemli) [inline templates/sshd_config.j2]: Something along the lines:

```
{% if sshd_listen_addresses|d() %}
{%   for address in sshd_listen_addresses %}
ListenAddress {{ address }}
{%   endfor %}
{% endif %}
```

**Comment by t2d** [comment]
yes, the list approach is nicer but not backward compatible. Therefore, I took a compatible approach that suits our current needs and the majority of use-cases.

**Comment by doobry-systemli** [comment]
Agreed, keeping backwards compatibility is important. We could keep using the old variable name `sshd_listen_address` (without `es` at the end). This would keep backwards compability as iterating over a string variable will treat it as a list with only one item.

**Comment by t2d** [comment]
Yes, this would also be fine with me. Go for it :)

**Comment by doobry-systemli** [comment]
Ready for review :)

**Pull request merged**